src/main/java/com/securian/creditcompass/login/LoginInteractor.java

```diff
@@ -0,0 +1,40 @@
+package com.securian.creditcompass.login;
+
+import com.securian.creditcompass.entities.ClaimsExaminer;
+import org.springframework.stereotype.Service;
+
+import javax.naming.AuthenticationException;
+import java.util.Optional;
+
+@Service
+public class LoginInteractor implements LoginInputBoundary{
+    private final LoginDataAccessInterface loginDataAccessInterface;
+    public LoginInteractor(LoginDataAccessInterface loginDataAccessInterface){
+        this.loginDataAccessInterface = loginDataAccessInterface;
+    }
+
+    @Override
+    public boolean authenticate(LoginInputData loginInputData) throws AuthenticationException{
+        // Create a ClaimsExaminer object from the username's equivalent in the database'
+        Optional<ClaimsExaminer> optionalExaminer = loginDataAccessInterface.findByUsername(loginInputData.getUsername());
+        if (optionalExaminer.isPresent()){
+            ClaimsExaminer examiner = optionalExaminer.get();
+            // get the password from the ClaimsExaminer object
+            String dbPassword = examiner.getPassword();
+
+            // compare database value to the password in the loginInputData
+            if(dbPassword.equals(loginInputData.getPassword())){
```

Comment:
Styling, there should be white space between the condition and the "if" keyword. Also I believe there should be whitespace before the first curly brace. I have made comments on other lines where I noticed it as well.